### DISCUSSION TITLE
manually paused project (free version) takes long time to pause

### DISCUSSION THREAD

**Discussion by webdev1hcc** [supabase/supabase#17545] [General]
I tried to manually pause my project in the Project Settings.  It is several hours now and it is still pausing.  Any ideas on how to stop the process or reset the project.  There is very little data in the project so I don't mind losing all of it.

**Comment by Tsimpliarakis** [comment]
How did that go? I'm pausing for the last 1 hour and i really need this to finish so i can continue working on the project..

**Comment by autopaybyphone** [comment]
Has this received any attention? Our project is also pausing for more than a week now.

This is a pain, since we cannot even delete it

**Comment by JoaquinVeraOrtega** [comment]
Happened to me too, was your project working correctly before the pause?

**Comment by shayanabbas** [comment]
it happening to me as well :S and its 2025 now so its a long lasting issue

**Comment by adilmohak** [comment]
It's been over a week and my project is still stuck in the "pausing" state after trying to pause it manually. This issue has been around for over a year now, and it's really disappointing to see that it still hasn't been addressed. Multiple developers have run into the same problem, yet there's been no clear resolution or update. Would appreciate if the team could acknowledge and prioritize this.

**Reply by JoaquinVeraOrtega** [reply]
Contact support. They gave me this answer: "It seems that the unresponsiveness of the project was due to an underlying hardware failure, which also caused pausing operations not to go through as to not generate data loss.

We've operated a migration for your project and is available once more.

In the future, we suggest operating a project restart, rather than pausing it, as restarting similarly results in a migration operation."
(And it worked btw)

**Comment by ohil24** [comment]
I got a response and I am not happy with it :
"Hi there,
Thanks for reaching out to us. We have received your support request and your ticket id is X.
Free plan will receive no guaranteed support response - we try to respond to outage related issues asap however Pro and Enterprise tickets take priority."

So what ? It's pushing me to upgrade to continue...

**Reply by eduhalfen** [reply]
Did you get a solution? Same thing occurs with me. My project sucks in pausing more than a day.

**Reply by ohil24** [reply]
No...

**Comment by Adam01-i** [comment]
I have the same probem my project stuck in the Pausing state for several hours.
I cannot delete or resume it via the dashboard or the CLI.
The error I get when trying to delete with CLI is:Failed to delete project <project_ref>: {"message":"Project is pausing and is not ready for deletion."}
I already tried:

Using supabase projects resume <project_ref>

Logging out and back in

Clearing browser cache

Using another browser and incognito mode

No luck so far.

**Comment by umputun** [comment]
This has been addressed in PR #349. The bot now properly handles `/report@botname` commands - it validates the bot username and only responds when it matches. Plain `/report` continues to work as before.
